src/pages/MonthlyGood/MonthlyGoodSetupPage.vue

```diff
@@ -392,6 +399,7 @@ export default {
 				});
 		},
 		result({ data }) {
+			this.isMGTaglineActive = _get(data, 'ui.mg_donationtagline_active.value') === 'true' || false;
```

Comment:
and make this line:
`this.isMGTaglineActive = _get(data, 'general.mgDonationTaglineActive.value') === 'true' || false;`